.github/workflows/preview-env-deploy.yml

```diff
@@ -28,21 +28,14 @@ jobs:
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_BUCKET_NAME;
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_GCLOUD_SA_KEY;
 
-      - name: Find previous deployment or tear-down comment
-        uses: peter-evans/find-comment@v3
-        id: find-comment
-        with:
-          issue-number: ${{ github.event.number }}
-          body-includes: <!-- preview-env -->
-
       - name: Upsert comment with build status
-        uses: peter-evans/create-or-update-comment@v4
+        uses: mshick/add-pr-comment@v2
         with:
-          comment-id: ${{ steps.find-comment.outputs.comment-id }}
-          issue-number: ${{ github.event.number }}
-          body: |
-            :construction: The preview environment for the commit ${{ github.sha }} is being built. This usually takes 15-20 minutes. <!-- preview-env -->
-          edit-mode: replace
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message-id: preview-env-status
```

Comment:
`message-id` is also not necessary, if you don't plan on having more than one comment you want to update throughout the process. There's a default value like `add-pr-comment`.